lib/api/metadata/index.ts

```diff
@@ -0,0 +1,45 @@
+import { queryAPI } from "@/lib/fetch";
+import { getSiteFromLocale } from "@/lib/helpers/site";
+import { fallbackLng } from "@/lib/i18n/settings";
+import { gql } from "graphql-request";
+
+export async function getEntryMetadataByUri(
```

Comment:
create a small fetch utility for getting the title/description for pages to use in metadata